run.sh

```diff
@@ -41,6 +41,7 @@ function build_image() {
 #    To rebuild a framework's container image for testing during local development, use build_image "$language/$framework"
 function run_test() {
   validate_environment
+  cleanup_tables
```

Comment:
Did you have a need to do cleanup prior to running?  If so, I'm reasonably sure this won't cause any issues, just want to make sure it's solving something :)

Reply:
This change seems to have been from @Akilan1999, he'll know more about it.

Reply:
It was a local change that accidentally got pushed. Removed in the last commit.